sources/MAKEINIT

```diff
@@ -24,22 +26,22 @@ Copyright (c) 1982-1988, 1990-1992, 1998, 2021 by Venue & Xerox Corporation.  Al
                      LLARITH LLFLOAT)
               (FILES RENAMEFNS XCL-PACKAGE CMLARRAY-SUPPORT VMEM))
         (COMS 
-              (* ;; "From original MAKEINIT")
+              (* ;; "From original MAKEINIT")
 
               (FNS LOADMAKEINIT LOADMKIFILES RELOAD MAKEINIT MKI.START)
-              (COMS                                          (* ; 
-                                     "reading compiled files and processing well-known expressions")
+              (COMS                                          (* ; 
```

Comment:
Ron, do you have something in your init that changes the fontset selection? The diffs seem to be full of changes that are ignored fontshifts